### DISCUSSION TITLE
How do I search Filenames by  wildcard

### DISCUSSION THREAD

**Discussion by gluizpessin** [tobya/DocTo#196] [Q&A] [answered]
Is there a way to utilize wildcards like * and ? in order to batch convert files?

**Comment by tobya** [comment]
Could you give me an example of what you are trying to do?

**Reply by gluizpessin** [reply]
Yes! I am setting up a Task in Scheduler to run a .bat to convert spreadsheets with specific names but all of them start with a default word, ie: Excel+(001).xlsx. 

From what I have tried, using * to substitute "+(001)" didn't work. Is there a alternative?

**Comment by tobya** [comment, marked as answer]
I have created an issue, I will do this in next release

**Comment by tobya** [comment]
There is a bit of a hack work around that works.  You can use -fx paramter

eg

docto -XL -f "c:\dir\withfiles" -o c:\temp\ -fx excel*.xls -t wdformatpdf

Thsi should find all files in c:\dir\withfiles that match the pattern excel*.xls

I'll see if I can put in a nicer syntax

**Comment by tobya** [comment]
Fix available and updated binary  at link below

- https://github.com/tobya/DocTo/issues/200

@gluizpessin I have added a new paramater

--inputfilter excel*.xls

if you use this in combination with a -f directory it should do what you want.

docto -f c:\dir\withfiles -o c:\outputfiles\ --inputfilter excel*.* -t = wdformatpdf

I have attached a new binary until I realease a new version.

Download here: [docto.zip](https://github.com/tobya/DocTo/files/9569862/docto.zip)
